quickwit/quickwit-control-plane/src/debouncer.rs

```diff
@@ -119,18 +119,23 @@ impl Debouncer {
     pub fn self_send_with_cooldown<M>(
         &self,
         ctx: &ActorContext<impl Handler<M> + DeferableReplyHandler<M>>,
+        message: M,
```

Comment:
```suggestion
        message: M,
        ctx: &ActorContext<impl Handler<M> + DeferableReplyHandler<M>>,   
```

we tend to make ctx the last argument